package.xml

```diff
@@ -1,22 +1,28 @@
 <?xml version="1.0"?>
-<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>ur_msgs</name>
   <version>1.3.4</version>
   <description>Message and service definitions for interacting with Universal Robots robot controllers.</description>
-  <author>Andrew Glusiec</author>
-  <author>Felix Messmer</author>
   <maintainer email="g.a.vanderhoorn@tudelft.nl">G.A. vd. Hoorn</maintainer>
   <maintainer email="miguel.prada@tecnalia.com">Miguel Prada Sarasola</maintainer>
   <maintainer email="nhg@ipa.fhg.de">Nadia Hammoudeh Garcia</maintainer>
   <license>BSD-3-Clause</license>
+  <author>Andrew Glusiec</author>
+  <author>Felix Messmer</author>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>rosidl_default_generators</buildtool_depend>
 
-  <build_depend>message_generation</build_depend>
+  <depend>builtin_interfaces</depend>
   <depend>geometry_msgs</depend>
-  <depend>std_msgs</depend>
-  <exec_depend>message_runtime</exec_depend>
+
+  <exec_depend>rosidl_default_runtime</exec_depend>
+
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
+
+  <member_of_group>rosidl_interface_packages</member_of_group>
 
   <export>
     <architecture_independent />
```

Comment:
Are ROS 2 msg pkgs still `architecture_independent`? They're not header-only any more, so I would not expect that to still be the case.

Reply:
Yes, thanks for spotting that. Fixed in 7ea5b71